maybe/syscall_filters.py

```diff
@@ -285,6 +310,30 @@ def substitute_dup(file_descriptor_old, file_descriptor_new=None):
     format=lambda args: format_open(args[1], args[2]),
     substitute=lambda args: substitute_open(args[1], args[2])
 ),
+SyscallFilter(
+    name="mknod",
+    signature=("int", (("const char *", "pathname"), ("mode_t", "mode"),)),
+    format=lambda args: format_mknod(args[0], args[1]),
+    substitute=return_zero
```

Comment:
When we return a value other than `None`, the syscall will be suppressed. This should only happen if we log the call, which we don't in all cases (compare `format`/`substitute_open`).